.travis.yml

```diff
@@ -0,0 +1,40 @@
+# We set the language to c because python isn't supported on the MacOS X nodes
+# on Travis. However, the language ends up being irrelevant anyway, since we
+# install Python ourselves using conda.
+language: c
+
+os:
+    - linux
+
+# Use Travis' container-based architecture
+sudo: false
+
+env:
+    global:
+        # The following versions are the 'default' for tests, unless
+        # overidden underneath. They are defined here in order to save having
+        # to repeat them for all configurations.
+        - PYTHON_VERSION=3.6
+        - PYTEST_VERSION=3.2
```

Comment:
Well, it seems you need numpy after all, at least here it's imported: https://github.com/astropy/pytest-doctestplus/blob/3dbb8b7f71506d4900c01f806dc54c80102a3cb8/pytest_doctestplus/output_checker.py#L10

Reply:
Whoops, yes I missed that one.